file_formats/ie_formats/ini_anim.htm

```diff
@@ -0,0 +1,849 @@
+---
+title: "INI file format: Creature Animations"
+---
+    <div class="title_main">INI file format: Creature Animations</div>
+
+    <br />
+    <div class="fileHeader">Applies to:</div>
+    <div class="indent1">BGEE</div>
```

Comment:
If they deviated enough not to mention PST, at least add it to the intro paragraph with a link to resdata.ini. It looks like it fell through the cracks otherwise.